airflow/configuration.py

```diff
@@ -1260,7 +1483,8 @@ def _deprecated_value_is_set_in_config(
             if config_type == "default":
                 continue
             try:
-                deprecated_section_array = config.items(section=deprecated_section, raw=True)
+                section_items = config.items(section=deprecated_section, raw=True)
+                deprecated_section_array = section_items
```

Comment:
Why this change?

Reply:
Remnant of  some other approach or debugging. bringing it back to original state